docs/content/using/csi_ug_using_sample.md

```diff
@@ -20,20 +20,20 @@ The following are examples of different types of secret names that can be given
 
 Use this information to run a stateful container on StatefulSet volumes using either file systems or raw block volumes.
```

Comment:
I can't directly comment above this line, but I would remove all the text from `While these examples specify ...` until this line (this line is fine)

Reply:
@oriyarde  I think it's important to note that the example is for SVC specifically. we originally had multiple samples up and we decided to merge into one but it's important to say that this is for sample purposes only and what we are using for it. 
FYI @barakda

Reply:
I don't see anything about this example which makes it specifically for SVC.
anyway, the main part which I think should be removed is the secret naming suggestions table

Reply:
@oriyarde  you are correct. I can remove it - but should it be placed in the secret configuration area or elsewhere instead?
at some point we had everything in the file but now we link out so it's not necessary but maybe we need to place the info elsewhere before just deleting.

Reply:
I think the naming suggestions table should be removed because I don't see a point in having it at all

Reply:
DONE

Reply:
I still see said table
![image](https://user-images.githubusercontent.com/57443811/134350774-d282659b-936a-4446-bf2f-0bc0bc9f5f0d.png)

Reply:
I know -from a documentation point of view I think that these should stay.

Reply:
we don't suggest how to name any other type of object, I don't see how secrets are any different.
each user should decide what is best for them, our docs should stick to what is relevant specifically for our CSI implementation.
this is both noise to the user and additional "mass" for us to maintain

Reply:
I really dont see the point of the naming samples, the secret name will not affect the driver operation so I cant understand why do we care..

Reply:
DONE

Reply:
IMO,
```
```

Reply:
I think that this can be left here as a best practice type of note.

Reply:
not only do I think users with multiple types of our storage systems are too rare to justify this, I'm also not aware of such best practice, nor do I think this practice is important in general

Reply:
@erantzabari  what do you think? should I just remove the note or leave in?

Reply:
I think it can be left as recommendation and we should change "be sure.." to "it is recommended..."

Reply:
Thanks, @erantzabari  per our conversation, i'll update to indicate that it should be done to  avoid confusion and help debugging but without the word "recommend" (but also not as a must), per the IBM Style guide.

Reply:
DONE